docs/explanation/lex-persona.md

```diff
@@ -0,0 +1,628 @@
+    "stepType" : "signature",
+    "recipients" : [ {
+      "consentPageId" : "cop_NzBEGbgTkj7v5************",
+      "email" : "<signer_email_address>@mail.com",
+      "country" : "FR",
+      "phoneNumber":"+33 7 ** ** ** **",
+      "preferredLocale" : "fr",
+      "firstName":"John",
+      "lastName":"Doe"
+    } ],
+    "requiredRecipients" : 1,
+    "validityPeriod" : 8553600000,
+    "invitePeriod" : 86400000,
+    "maxInvites" : 5,
+    "sendDownloadLink" : true,
+    "allowComments" : true,
+    "hideAttachments" : false,
+    "hideWorkflowRecipients" : true
+  } ],
+  "notifiedEvents" : [ "recipientRefused", "recipientFinished", "workflowStopped", "workflowFinished" ],
+  "watchers" : []
+}
+```
+Response :
+
+```json
+{
+    "created": 1688993660520,
+    "currentRecipientEmails": [],
+    "currentRecipientUsers": [],
+    "description": "91 Allée, Voie de Presbourg, 60282 Vénissieux",
+    "email": "<user_email>@mail.fr",
+    "firstName": "Sam",
+    "groupId": "grp_9woWtcSt5KiAzyq************",
+    "id": "wfl_4yzLat7tDUQ8Jw************",
+    "lastName": "P******",
+    "logs": [],
+    "name": "Aerodynamic Silk Bottle",
+    "notifiedEvents": [
+        "recipientRefused",
+        "recipientFinished",
+        "workflowStopped",
+        "workflowFinished"
+    ],
+    "progress": 0,
+    "steps": [
+        {
+            "allowComments": true,
+            "hideAttachments": false,
+            "hideWorkflowRecipients": true,
+            "id": "stp_D5KgBHk63MkhdGw********",
+            "invitePeriod": 86400000,
+            "isFinished": false,
+            "isStarted": false,
+            "logs": [],
+            "maxInvites": 5,
+            "recipients": [
+                {
+                    "consentPageId": "cop_NzBEGbgTkj7v5tq********",
+                    "country": "FR",
+                    "email": "<signer_email_address>@mail.com",
+                    "firstName": "John",
+                    "lastName": "Doe",
+                    "phoneNumber": "+33 7 ** ** ** **",
+                    "preferredLocale": "fr"
+                }
+            ],
+            "requiredRecipients": 1,
+            "sendDownloadLink": true,
+            "stepType": "signature",
+            "validityPeriod": 8553600000
+        }
+    ],
+    "tenantId": "ten_9H4TvNNQFfrLBXH********",
+    "updated": 1688993660520,
+    "userId": "usr_P5YBCEeRqY8jKGG********",
+    "viewAuthorizedGroups": [
+        "grp_9woWtcSt5KiAzyq********"
+    ],
+    "viewAuthorizedUsers": [],
+    "watchers": [],
+    "workflowStatus": "stopped"
+}
+```
+Once, this is done, you have just created the workflow, well done !
+
+In this example, I've decided to send the document to be signed to a user that is not yet registered on the Lex Persona BackOffice.
+
+This example shows you that it is not required to create a "user".
+We can just parse the signer's details into the "recipients" lists of the payload (firstName, lastName, phoneNumber, email).
```

Comment:
**When the signer is an external contact in LexPersona :** 

It is required to send the `firstName` and the `lastName` of the signer if she/he is an external contact.  If one of those fields is missing, the API responds with a Bad request - 400 : "A recipient in the request is missing identity information." It also responds with a Bad request if the `email` is missing.

If we provide a `userId`, we do not need to send the `firstName`, `lastName`, `email` and `phoneNumber` since this data is saved into the user's object (this example requires to create a user for each signer).

If we do not provide a `userId` , the required data of a contact are the following ones: 
- With SMS validation : 
    - `firstName`, `lastName`, `email`, `phoneNumber`

- Without SMS validation : 
    - the `phoneNumber` is not required.

Reply:
Info : If we only put the `name` as a full name in the payload instead of `firstName` and `lastName`, the API responds with a 400 - Bad Request.